scaleway/resource_instance_snapshot.go

```diff
@@ -36,7 +36,7 @@ func resourceScalewayInstanceSnapshot() *schema.Resource {
 			},
 			"volume_id": {
 				Type:         schema.TypeString,
-				Required:     true,
+				Optional:     true,
```

Comment:
Is it possible to snap without volumeID?  As long as your example show yes, why is set on your request to create?

Reply:
Import of qcow is made without `volume_id`